azext_edge/edge/params.py

```diff
@@ -1260,6 +1260,88 @@ def load_iotops_arguments(self, _):
             "/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/roleDefinitions/{roleId}",
         )
 
+    with self.argument_context("iot ops connector opcua trust") as context:
+        context.argument(
```

Comment:
you can put shared params in "iot ops connector"

Reply:
I got your point, I will probably put under "iot ops connector opcua"